network-api/networkapi/wagtailpages/models.py

```diff
@@ -574,7 +582,6 @@ class BlogPageTag(TaggedItemBase):
 
 
 class BlogPage(FoundationMetadataPageMixin, Page):
-    template = 'wagtailpages/blog_page.html'
```

Comment:
as a normal page model (rather than page with a static template backing it) this line was not necessary.